airflow/config_templates/default_test.cfg

```diff
@@ -50,14 +50,8 @@ sql_alchemy_conn = sqlite:///{AIRFLOW_HOME}/unittests.db
 load_default_connections = True
 
 [logging]
-base_log_folder = {AIRFLOW_HOME}/logs
-logging_level = INFO
-celery_logging_level = WARN
-fab_logging_level = WARN
-log_filename_template = {{{{ ti.dag_id }}}}/{{{{ ti.task_id }}}}/{{{{ ts }}}}/{{{{ try_number }}}}.log
-log_processor_filename_template = {{{{ filename }}}}.log
-dag_processor_manager_log_location = {AIRFLOW_HOME}/logs/dag_processor_manager/dag_processor_manager.log
-worker_log_server_port = 8793
+celery_logging_level = WARNING
+fab_logging_level = WARNING
```

Comment:
Removing unnecessary defaults to eliminate some warnings at test run time.

Reply:
Can we do these as a separate PR please?